h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -13,11 +13,16 @@
 import static water.fvec.Vec.makeCon;
 
 public class SQLManager {
-  
-  final static String TEMP_TABLE_NAME = "table_for_h2o_import";
+
+  private final static String TEMP_TABLE_NAME = "table_for_h2o_import";
```

Comment:
General note (In this case it is okay): we need to be careful about changing the access modifiers H2O is oftentimes embedded in 3rd party product and making something private can cause issues for the users.

Reply:
Thank you. Noted :)